### PR TITLE
DOC: DataFrame.get_value

### PR DESCRIPTION
**Pull request by palewire**
Basic examples for working with DataFrame's ``get_value`` method. I do not understand how the ``takeable`` option is supposed to work. If someone can explain it to me, I'd be happy to add an example for that as well.

### CODE REVIEW

**Comment by jreback** [comment]
see #15269 ``takeable`` is an internal parameter (you can mark it this way). This method is slated for deprecation.

**Comment by palewire** [comment]
Thanks @jreback. I've just pushed an addition to the docstring that mirrors your language. Since it's an internal option that will be deprecated, I'm guessing we don't need an example that shows its use.

Let me know what else I can do to prepare this for a merge.

**Comment by jreback** [comment]
@palewire maybe I wasn't clear. This entire function is to be deprecated. But at the moment its not, so simply label it as *internal*

**Comment by palewire** [comment]
Ah. Now I understand. What alternative method would you recommend as the best replacement? I could also refer readers there.

**Comment by jreback** [comment]
@palewire this PR is about documentation, but NOT deprecation (that's the other issue).

``df.get_values(row, col)`` -> ``df.at[row, col]`` or ``df.iat[row, col]``

this is simply a functioal version of ``.at`` (or with the ``tableable``is ``.iat``

**Comment by palewire** [comment]
@jreback Alrighty. Thanks for clearing that up. I've just pushed some edits that clearly limit this as an internal method slated for deprecation. I've also appended a note that offers the recommended alternatives.

File: pandas/core/frame.py [hunk 1]
```diff
@@ -1856,17 +1856,58 @@ def _unpickle_matrix_compat(self, state):  # pragma: no cover
...
+        An internal method to quickly retrieve single value from the
+        provided column and index. Slated for deprecation.
 
         Parameters
         ----------
         index : row label
         col : column label
-        takeable : interpret the index/col as indexers, default False
+        takeable : Instructs the method to interpret the index
+        and col parameters as indexers. The default is ``False``.
```
Comment (jreback) [inline pandas/core/frame.py]: as positional indexers

File: pandas/core/frame.py [hunk 2]
```diff
@@ -1856,17 +1856,58 @@ def _unpickle_matrix_compat(self, state):  # pragma: no cover
...
 
         Returns
         -------
         value : scalar value
+
+        Notes
+        -----
+
```
Comment (jreback) [inline pandas/core/frame.py]: extra line

File: pandas/core/frame.py [hunk 3]
```diff
@@ -1856,17 +1856,58 @@ def _unpickle_matrix_compat(self, state):  # pragma: no cover
...
+        >>> df.get_value(0, 'object')
+        'a'
+        >>> df.get_value(1, 'numeric')
+        2
+        >>> df.get_value(1, 'object')
+        'b'
+
+        If the ``DataFrame`` has a non-numeric index you can use that as well.
+
+        >>> df.set_index('object', inplace=True)
```
Comment (jreback) [inline pandas/core/frame.py]: don't use inplace

**Comment by jreback** [comment]
can you update Series.get_value, set_value and DataFrame.set_value

**Comment by jreback** [comment]
can you update

**Comment by jreback** [comment]
can you update

**Comment by jreback** [comment]
can you update

**Comment by jreback** [comment]
closing as stale

**Pull request closed without merging**